### ISSUE TITLE
Upgrade to stable busybox 1.32.0

### ISSUE THREAD

**Issue by macmpi** [maxnet/berryboot#560]
Hi,

[Busybox 1.30](https://busybox.net/) brought many feature enhancements and bugfixes: in particular TLS support has been greatly improved, which now allows much better HTTPS `wget` downloads [compatibility](https://github.com/maxnet/berryboot/issues/439#issuecomment-439105472).

Any chance berryboot can move to latest stable busybox [1.30.1](https://github.com/maxnet/berryboot/blob/berryboot2018/buildroot-2018.08/package/busybox/busybox.mk)?
(note: in that move, [.config](https://github.com/maxnet/berryboot/blob/berryboot2018/buildroot-2018.08/package/busybox/busybox.config) files should be synced with up-to-date versions for consistency: it seems those configs are still on 1.28.x whereas currently used busybox is 1.29.x)

**Comment by ah584d** [comment]
Hi Guys i still meet the issue with wget and https:
wget: TLS error from peer (alert code 40): handshake failure
wget: error getting response: Connection reset by peer

I use version 1.30.1
I thought it was supposed to be fixed? any clue?

thanks

**Comment by macmpi** [comment]
Have you recompiled berryboot with updated busybox 1.30.1 and assorted config files? (current available berryboot binary is on earlier version).
Or are you using some pre-built busybox binaries from some other source? Config file needs to be properly set for TLS, which may not be the case in that binary?

As a test, can you successfully download this sample URL with busybox wget:
https://ftp.openbsd.org/pub/OpenBSD/LibreSSL/libressl-2.6.4.tar.gz
If so, you probably have a correct busybox 1.30.1 setup. If it fails, I'm afraid you may need to rebuild busybox with proper config.

If you have correct busybox, then maybe your destination URL server may require a cypher that is not currently implemented: can you share the failing URL to confirm?

**Comment by ah584d** [comment]
Hi, 
i did not recompiled anything, i just added in my pod YAML file 
- image: busybox:1.30.1
hoping that it will work
I can see that my pod download the correct image, but wget still does not work with https.
Can you please post a link with the explanation about the procedure you described? I'm not sure that i understand you, i thought that i just needed to download the last busybox and that it will work out of the box
Thanks

**Comment by macmpi** [comment]
No: @ah584d for now, you would need to rebuild berryboot from source by yourself, after making [a few changes to the build files](https://github.com/maxnet/berryboot/blob/berryboot2018/buildroot-2018.08/package/busybox/busybox.mk#L7) to enjoy new busybox version with enhanced wget/TLS support.

Maybe, if @maxnet elects to at some point, a later berryboot binary release might hopefully integrate that update.

**Comment by macmpi** [comment]
Just passed [1.31.1 stable](https://git.busybox.net/busybox/commit/?h=1_31_stable): thanks for considering integration in berryboot (and leverage better https support and other bug fixes).

**Comment by macmpi** [comment]
as time passes, is now [1.32.0 stable](https://git.busybox.net/busybox/log/?h=1_32_stable)
